embedding_reader/get_file_list.py

```diff
@@ -42,7 +42,7 @@ def _get_file_list(
     path = make_path_absolute(path)
     fs, path_in_fs = fsspec.core.url_to_fs(path)
     prefix = path[: path.index(path_in_fs)]
-    glob_pattern = path.rstrip("/") + f"**/*.{file_format}"
```

Comment:
this was definitely wrong

Reply:
path = "my_folder"
path.rstrip("/") + f"**/*.{file_format}"
-> 'my_folder**/*.parquet'